react_paragraphs.install

```diff
@@ -194,6 +194,11 @@ function react_paragraphs_update_8203(&$sandbox) {
     $rows = [];
     foreach ($entity->get($field_name)->getValue() as $field_item) {
       $field_item['settings'] = json_decode($field_item['settings'], TRUE);
+      // Because the serializer is gone, the settings might be a double encoded
+      // json string, so we will want to check to try and decode it again.
```

Comment:
huh. Very strange. I did not see that in my testing but I suppose this can't hurt.

Reply:
Me either until I ran it on the gryphon stack. I'm not sure if its the order of execution or maybe something else.